packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -279,12 +275,19 @@ class PouchLink extends CozyLink {
    * Emits an event (pouchlink:sync:end) when the sync (all doctypes) is done
    */
   handleOnSync(doctypeUpdates) {
-    const normalizedData = mapValues(doctypeUpdates, normalizeAll(this.client))
+    if (doctypeUpdates && Array.isArray(Object.keys(doctypeUpdates))) {
+      Object.keys(doctypeUpdates).forEach(doctype => {
```

Comment:
nit: We do two times the `Object.keys(doctypeUpdates)` call

Reply:
Good point, even though here the keys are the doctype names, so the impact should be $\epsilon$

Reply:
Fixed in [4db9158](https://github.com/cozy/cozy-client/pull/1596/commits/4db91588a3298ef0bb2040ce124fa0299482d52c)